migration-overview.md

```diff
@@ -6,63 +6,61 @@ aliases: ['/docs/dev/migration-overview/']
 
 # Migration Overview
 
-This document describes how to migrate data to TiDB, including migrating data from MySQL and from CSV/SQL files.
+This document describes how to migrate data to TiDB, which provides the following data migration features:
 
-## Migrate from Aurora to TiDB
+- Full data migration: use `TiDB Lightning` to import data(CSV/Aurora Snapshot/mydumper sql) to TiDB cluster; to better match with full migration from MySQL/MariaDB database, TiDB also provides a data export tool, `dumpling`, which supports exporting full data as CSV/mydumper sql format;
```

Comment:
```suggestion
- Full data migration:

   - To migrate data in a small volume (< 1 T), it is recommended that you use the TiDB DM tool. 
   - To migrate data of more than 1 T, it is recommended that you use the TiDB Dumpling tool to export full data in the CSV or mydumper SQL format, and then use the TiDB Dumpling tool to import the data. For Aurora, you can export the snapshot data of Aurora to Amazon S3, and then use the TiDB Dumpling tool to import the data.
```

Reply:
This is how I think about it. If the user needs incremental migration after the full migration, then the DM tool is recommended for the full migration of data less than 1T; otherwise, it is recommended to use lightning directly. For this intention, how do you think it is better to express it

Reply:
Here it is an overview of data migration features. I wonder if such expressions seem a little specific for an overview of features? Maybe it could be put in the sections that follow? @qiancai